content/blog/2019/cve-2019-12995/index.md

```diff
@@ -0,0 +1,96 @@
+A bug in Istio’s JWT validation filter causes Istio Proxy to crash in certain cases when the request contains a malformed JWT token. The bug was discovered and reported by a user [on github](https://github.com/istio/istio/issues/15084) on June 23, 2019.
```

Comment:
Needs front-matter.
Uses of triple-ticks must be replaced with `{{<text>}}`

Reply:
Not sure to understand, would you mind provide a suggestion?